app/livechat/client/views/app/tabbar/customerChatHistoryMessages.js

```diff
@@ -0,0 +1,101 @@
+import { ReactiveVar } from 'meteor/reactive-var';
+import { Template } from 'meteor/templating';
+import moment from 'moment';
+import _ from 'underscore';
+import './customerChatHistoryMessages.html';
+import { APIClient, t } from '../../../../../utils/client';
+import { AccountBox, TabBar, MessageTypes } from '../../../../../ui-utils';
```

Comment:
Again, a lot of unused imports.

Reply:
only required packages are importing now